.tree-sitter-lint/tree-sitter-lint-local/src/main.rs

```diff
@@ -1,11 +1,17 @@
 use std::sync::Arc;
 
-use tree_sitter_lint::{clap::Parser, Args, Rule};
+use tree_sitter_lint::{clap::Parser, Args, Plugin, Rule};
 
 fn main() {
-    tree_sitter_lint::run_and_output(Args::parse().load_config_file_and_into_config(all_rules()));
+    tree_sitter_lint::run_and_output(
+        Args::parse().load_config_file_and_into_config(all_plugins(), all_standalone_rules()),
```

Comment:
So the `Config`-generating method on `Args` now takes "all plugins" as well as "all standalone rules" as arguments

My instinct is that even though that feels a little weirdly asymmetrical it is nice for the `local_rules/` to be able to be different from a plugin: it just is expected to expose a function that returns a simple `Vec<Arc<dyn Rule>>` (as opposed to plugins being expected to return a `Plugin` which will in theory be able to include other things besides just rules) and its rule names don't get prefixed with a plugin name

Otherwise it feels like there would be some ergonomic downside to having to "treat the local rules like a plugin" (and then it becomes weird that it's a singleton - what if you want to have more than one "local plugin"? (with this approach the answer to that is "you just put them wherever you want locally (eg could be committed to your project repo if you want) and then declare them as `path`-style `plugins` in your config file"))